troposphere/static/js/components/projects/detail/resources/link/ExternalLinkTable.jsx

```diff
@@ -33,8 +33,9 @@ export default React.createClass({
             selectedResources = this.props.selectedResources;
 
         return external_links.map(function(external_link) {
-            var isPreviewed = (previewedResource === external_link),
-                isChecked = selectedResources.get(external_link) ? true : false;
+            let id = external_link.get("id"),
+                isPreviewed = (previewedResource === external_link),
+                isChecked = selectedResources.findWhere({id}) ? true : false;
```

Comment:
There may be some confusion here, the `ExternalLink` model was created after Django `orm` supported the UUID field type participating a primary key constraint. So with `ExternalLink`, there is no `"uuid"` attribute.